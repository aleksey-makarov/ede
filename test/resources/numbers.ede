{
  "double": 1.0567,
  "integral": 492,
  "positive": 10.0,
  "negative": -0.05
}
---
{{ double }}
{{ integral }}
{{ positive }}
{{ negative }}
