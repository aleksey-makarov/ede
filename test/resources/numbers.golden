1.0567
492
10
-0.05
